### PR TITLE
chore: add krankerl.toml and .nextcloudignore

### PR DESCRIPTION
**Pull request by edward-ly**
I'm not sure how much krankerl is still being used anymore, but I think it's still useful to have regardless if only for educational purposes.

The `.nextcloudignore` file in particular would be useful for me in the context of the [Hello World tutorial](https://cloud.nextcloud.com/s/iyNGp8ryWxc7Efa?path=%2F%2F2%20Develop%20your%20first%20Hello%20World%20app). For the **Delete files and directories we don't need** step, rather than select and delete each file individually, I'd be able to run this command instead:

```
rm -rf $(sed 's/^\/\?/.\//g' .nextcloudignore)
```

### CODE REVIEW

File: .nextcloudignore [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+/.gitattributes
+/.gitignore
+/.l10nignore
+/.nextcloudignore
+/.nvmrc
+/.php-cs-fixer.*
+/.pre-commit-config.yaml
+/.travis.yml
+/COPYING*
+/LICEN[CS]E*
```
Comment (provokateurin) [inline .nextcloudignore:17]:
```suggestion
```
These need to be included

File: .nextcloudignore [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+/[Mm]akefile
+/babel.config.js
+/build
+/composer.*
```
Comment (provokateurin) [inline .nextcloudignore:21]:
```suggestion
```
Need to be included

File: .nextcloudignore [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+/jest.config.js
+/krankerl.toml
+/makefile
```
Comment (provokateurin) [inline .nextcloudignore:24]:
```suggestion
```
Duplicate

File: .nextcloudignore [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+/node_modules
+/openapi.json
```
Comment (provokateurin) [inline .nextcloudignore:26]:
```suggestion
```
Need to be included (and the pattern wasn't even matching all of them).

File: .nextcloudignore [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+/package*.json
```
Comment (provokateurin) [inline .nextcloudignore:27]:
```suggestion
```
Need to be included

**Comment by edward-ly** [comment]
> I'm not sure if this is a good idea. The template is meant for community purposes while krankerl is only used in the company (at least to my knowledge).

WIth [the number of downloads it has](https://crates.io/crates/krankerl) and the fact that it is FOSS, I imagine there should be at least some community usage out there, but I'm not familiar with the software myself, so I could be wrong too.

Anyway, I'm still closing this because I see now that the files to be ignored in the Hello World app don't apply to all apps in general. I don't think the list is worth adding anymore either if only for the fact that it won't cover everything that is to be deleted for the tutorial. Thanks for the insight.

**Comment by provokateurin** [comment]
Krankerl is used in CI which probably explains all the installs you are seeing in the graph.

**Pull request closed without merging**